scripts/http-jsonp-detection.nse

```diff
@@ -50,8 +51,7 @@ local callbacks = {"cb", "jsonp", "jsonpcallback", "jcb", "call"}
 --Checks the body and returns if valid json data is present in callback function
 local checkjson = function(body)
    
-  local func, json_data
-  _, _, func, json_data = string.find(body, "(%S+)%((.*)%)")  --Check this!
+  local _, _, func, json_data = string.find(body, "(%S+)%((.*)%)")  --Check this!
```

Comment:
Ignore the warnings about _ by nmap_check. It is a known limitation right now. It seems it doesn't recognize/treat the character as it is supposed to.